bin/clean-frontend.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
-sed -i 's/\/ember/http:\/\/localhost:4200\/app/g' app/views/layouts/application.html.haml
-vendorjs="$(grep -oP '(?<=<script src="\/app\/assets\/)vendor-[a-z|\d]{32}.js' public/ember/index.html)"
-vendorcss="$(grep -oP '(?<=<link integrity="" rel="stylesheet" href="\/app\/assets\/)vendor-[a-z|\d]{32}.css' public/ember/index.html)"
-frontendjs="$(grep -oP '(?<=<script src="\/app\/assets\/)frontend-[a-z|\d]{32}.js' public/ember/index.html)"
-frontendcss="$(grep -oP '(?<=<link integrity="" rel="stylesheet" href="\/app\/assets\/)frontend-[a-z|\d]{32}.css' public/ember/index.html)"
-sed -i "s/$vendorjs/vendor.js/g" app/views/layouts/application.html.haml
-sed -i "s/$vendorcss/vendor.css/g" app/views/layouts/application.html.haml
-sed -i "s/$frontendjs/frontend.js/g" app/views/layouts/application.html.haml
-sed -i "s/$frontendcss/frontend.css/g" app/views/layouts/application.html.haml
+sed -i 's/\/ember/http:\/\/localhost:4200/g' frontend/app/index.html
+vendorjs="$(grep -oP '(?<=<script src="assets\/)vendor-[a-z|\d]{32}.js' public/ember/index.html)"
+vendorcss="$(grep -oP '(?<=<link integrity="" rel="stylesheet" href="assets\/)vendor-[a-z|\d]{32}.css' public/ember/index.html)"
+frontendjs="$(grep -oP '(?<=<script src="assets\/)frontend-[a-z|\d]{32}.js' public/ember/index.html)"
+frontendcss="$(grep -oP '(?<=<link integrity="" rel="stylesheet" href="assets\/)frontend-[a-z|\d]{32}.css' public/ember/index.html)"
+sed -i "s/$vendorjs/vendor.js/g" frontend/app/index.html
+sed -i "s/$vendorcss/vendor.css/g" frontend/app/index.html
+sed -i "s/$frontendjs/frontend.js/g" frontend/app/index.html
+sed -i "s/$frontendcss/frontend.css/g" frontend/app/index.html
```

Comment:
can't we just deliver the right index.html instead of replacing those entries ? 
this was required when ember was partially integrated ... but now ?

check sweb !

Reply:
But can we really do that? Thing is, when developing, we need to access the assets from the frontend server and when in prod, we need to deliver the built assets. But we always access the backend server. Meaning, we would need to switch between the index.html (+plus all the assets) the dev server builds and the index.html we build during the post-assemble. So it's not impossible I guess, it just seems like a big hustle to me. Especially since we'll change it again once we moved everything to ember, at that point we can do it exactly like sweb. Or am I overseeing something? :thinking: